server/src/organizations/dto/return-organization.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { IsNotEmpty } from 'class-validator';
+
+import { CreateOrganizationDto } from './create-organization.dto';
+
+export class ReturnOrganizationDto extends CreateOrganizationDto {
+  @IsNotEmpty()
+  id: number;
+}
```

Comment:
I think we should define the properties here instead of extending from `createDto..`. It should be most properties defined in the entity, except any sensitive ones.

Reply:
@esteban-gs, my thought was that data in the `createDto..`, the properties correspond to data provided by the user and so can be exposed, but I see that there might be user values we want to hide.  I was thinking the following properties could be sensitive: `nonprofit_classification` and `ein`.  Are there other properties we should be excluding in the response?